futurenhs.api/FutureNHS.Api/Services/Validation/FolderValidator.cs

```diff
@@ -25,13 +25,13 @@ public FolderValidator(IFolderCommand folderCommand)
 
             RuleFor(model => model)
                 .MustAsync(IsFolderUnique)
-                .OverridePropertyName(nameof(FolderDto.Title))
+                .OverridePropertyName(nameof(FolderDto.Name))
                 .WithMessage("Enter a unique folder title");                
```

Comment:
should we change the message to "Enter a unique folder name"?

Reply:
Good spot! will change this